openmc/source.py

```diff
@@ -104,6 +104,14 @@ def strength(self, strength):
         self._strength = strength
 
     def to_xml(self):
+        """Return XML representation of the source
```

Comment:
As an aside, I like the succinct `to_xml()` over our legacy `export_to_xml()` - what do you think about making this the standard?